File: modul2/Python/PRAK205_2310817210017_MUHAMMADNURHUDABAIHAQI.py
import math
tinggi, miring = map(int, input().split())
alas = int (math.sqrt(miring*miring-tinggi*tinggi))
luas = int(0.5*alas*tinggi)
keliling = int(alas+tinggi+miring)
print(f"Alas = {alas} cm")
print(f"Tinggi = {tinggi} cm")
print(f"Keliling = {keliling} cm")
print(f"Luas = {luas} cm^2")

File: modul2/C/PRAK205_2310817210017_MUHAMMADNURHUDABAIHAQI.c
#include <stdio.h>
#include <math.h>
int main() 
{
    int tinggi, miring, alas, keliling, luas;
    printf("");
    scanf("%d", &tinggi);
    printf("");
    scanf("%d", &miring);

    alas = sqrt(miring*miring-tinggi*tinggi);
    keliling = (alas+tinggi+miring);
    luas = (0.5*alas*tinggi);
    printf("Alas = %.d cm \n", alas);
    printf("Tinggi = %.d cm \n", tinggi);
    printf("Keliling = %.d cm \n", keliling);
    printf("Luas = %.d cm^2 \n", luas);

    return 0;
    
} 